llama.cpp

```diff
@@ -8138,6 +8141,7 @@ struct llm_tokenizer_bpe {
                 const auto token = vocab.token_to_id.find(str);
 
                 if (token == vocab.token_to_id.end()) {
+                    output.reserve(str.end() - str.begin());
```

Comment:
This isn't quite right - output isn't necessarily empty, we're just appending to it. And reserving in a loop is not going to do the right thing - best to just remove this.